packages/checkout-extension/src/ExtensionService.ts

```diff
@@ -22,6 +22,35 @@ export class ExtensionService {
         await this.checkoutService.loadExtensions();
     }
 
+    preloadExtensions(): void {
+        const state = this.checkoutService.getState();
+        const extensions = state.data.getExtensions();
+        const cartId = state.data.getCart()?.id;
```

Comment:
Can we use `getCartOrThrow` here?

Reply:
I don't think that method is available in state data `CheckoutStoreSelector`.